nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/tooltips/nifi-tooltip.directive.ts

```diff
@@ -58,11 +59,29 @@ export class NifiTooltipDirective<T> implements OnDestroy {
         }
     }
 
+    @HostListener('mousemove')
+    mouseMove() {
+        if (this.overlayRef?.hasAttached() && this.tooltipDisabled) {
+            this.overlayRef?.detach();
+        } else {
+            this.attach();
```

Comment:
I don't think this attach is needed. In fact it causes console errors...

<img width="803" alt="Screenshot 2024-05-29 at 8 49 10 AM" src="https://github.com/apache/nifi/assets/713866/e9060af1-ddb5-44f5-9e24-9089513668e4">